src/eos-updater-apply.c

```diff
@@ -78,7 +78,6 @@ apply (GTask *task,
   GMainContext *task_context = g_main_context_new ();
   const gchar *update_id = eos_updater_get_update_id (updater);
   const gchar *update_refspec = eos_updater_get_update_refspec (updater);
-  const gchar *orig_refspec = eos_updater_get_original_refspec (updater);
```

Comment:
I'll be honest that I originally did it because I wasn't sure the best way to store private data and that was the fastest way to make the property available. However, I do think it's useful for the client to see that this has changed, and it's also useful for debugging.